src/python/competition.py

```diff
@@ -107,7 +107,9 @@ def __init__(self, checkout, dataSource,
                maxConcurrentMerges = 1,  # use 1 for spinning-magnets and 3 for fast SSD
                addDVFields = False,
                name = None,
-               indexSort = None
+               indexSort = None,
+               vectorFile = None,
```

Comment:
Looks like `nightlyBench.py` will use these defaults, so it will not test vectors at first (with this PR).  I'll try to open a follow-on PR to add them to nightly, and to make a new chart -- a new nightly chart is born (soon)!